src/app/beer_garden/api/http/handlers/authorization_handler.py

```diff
@@ -200,8 +204,13 @@ def _get_token_payload_from_request(self) -> dict:
             token = auth_header.split(" ")[1]
             algorithm = jwt.get_unverified_header(token)["alg"]
 
-            return jwt.decode(token, key=secret_key, algorithms=[algorithm])
+            decoded_token = jwt.decode(token, key=secret_key, algorithms=[algorithm])
         except (jwt.InvalidSignatureError, jwt.DecodeError, KeyError, IndexError):
             raise InvalidToken
         except jwt.ExpiredSignatureError:
             raise ExpiredToken
+
+        if decoded_token["type"] != "access":
+            raise InvalidToken
```

Comment:
The *Raises* portion of the docstring should be updated with `InvalidToken` and `ExpiredToken`.

Reply:
Done.